cmake/FindCBLAS.cmake

```diff
@@ -0,0 +1,74 @@
+# ==================================================================================================
```

Comment:
Maybe worth adding an ack to here https://github.com/PennyLaneAI/pennylane-lightning/#acknowledgements

Reply:
I guess he meant that you should consider adding the acknowledgment to these contributions to the respective section on the readme file.

Reply:
Forgot to update this. After speaking with @mlxd , I think the original contributor and project (Cedric Nugteren <www.cedricnugteren.nl>) are already acknowledged (or did I miss the point?).